packages/edit-site/src/components/editor/utils.js

```diff
@@ -0,0 +1,33 @@
+/* CSS properties */
+export const FONT_SIZE = 'font-size';
+export const LINK_COLOR = '--wp--style--color--link';
+export const BACKGROUND_COLOR = 'background-color';
+export const GRADIENT_COLOR = 'background';
+export const TEXT_COLOR = 'color';
+export const LINE_HEIGHT = 'line-height';
+
+/* Supporting data */
+export const GLOBAL_CONTEXT = 'global';
+export const PRESET_CATEGORIES = [ 'color', 'font-size', 'gradient' ];
+export const STYLE_PROPS = {
```

Comment:
This mappings are also here https://github.com/WordPress/gutenberg/blob/2e639e45705ecf706c9c854133b493fd0a4ac0cb/packages/block-editor/src/hooks/style.js#L55 and here https://github.com/WordPress/gutenberg/blob/379719fe72699b0e2077d8263f5d78a0e3eecd33/lib/global-styles.php#L385.

I guess instead of repeating the mappings all over again we could take this opportunity to make the mappings available in a single place? Maybe on the server and for now passed to the client using a setting?

Reply:
I agree. The duplication predates this PR, though, and it seems to me that this is the kind of thing that would make a nice and isolated follow-up PR. I went ahead and created https://github.com/WordPress/gutenberg/issues/25051 for reference so we don't forget.